workspaces/marketplace/plugins/marketplace/src/components/MarketplacePluginContent.tsx

```diff
@@ -172,11 +183,36 @@ const PluginPackageTable = ({ plugin }: { plugin: MarketplacePlugin }) => {
   );
 };
 
+const ActionsMenu = (props: MenuProps) => (
+  <Menu
+    elevation={0}
+    anchorOrigin={{
+      vertical: 'bottom',
+      horizontal: 'left',
+    }}
+    transformOrigin={{
+      vertical: 'top',
+      horizontal: 'left',
+    }}
+    {...props}
+  />
+);
+
```

Comment:
Good to abstract this component to a different file as the parameter is only passed in `props`.